C/casacorewrapper/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "casacorewrapper"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kiranshila/casascorewrapper.git", "d7574e8639c8ac55a360652e4e799a4686bec93c")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/casascorewrapper/
+mkdir build && cd build
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCASACORE_ROOT_DIR=${prefix} \
+    ..
+make -j${nproc}
+make install
+"""
+
+# Use the same platforms from casacore
+platforms = supported_platforms(exclude=(platform)-> Sys.iswindows(platform) || Sys.isfreebsd(platform))
+platforms = expand_cxxstring_abis(platforms)
+# Expand libgfortran versions because we need to exclude some specific combinations
+platforms = expand_gfortran_versions(platforms)
+filter!(p -> libc(p) != "musl", platforms)
+filter!(p -> !(arch(p) == "powerpc64le" && libgfortran_version(p) == v"3"), platforms)
```

Comment:
Also this line can be removed
```suggestion
```

Reply:
I tried that, but it was saying it couldn't find artifacts for powerpc64le-linux-gnu-libgfortran3

Reply:
I'll try again